app.py

```diff
@@ -71,10 +71,10 @@ def register():
               
 
                 test_response = requests.post(URL+'registerNetwork', json={'email_field': email, 'host_url_field': request.url_root})   
-                if test_response.ok:
-                    flash("host has been registered")
-                else:
```

Comment:
instead of commenting out the entire if else clause, 
just instead replace the line that return "error", to something like flashing error.

Reply:
ohk i will do that